build.psm1

```diff
@@ -191,51 +167,48 @@ function Start-ScriptAnalyzerBuild
 
         $destinationDir = "$projectRoot\out\PSScriptAnalyzer"
         # this is normalizing case as well as selecting the proper location
-        if ( $Framework -eq "core" ) {
-            $destinationDirBinaries = "$destinationDir\coreclr"
-        }
-        elseif ($PSVersion -eq '3') {
+        if ($PSVersion -eq '3') {
             $destinationDirBinaries = "$destinationDir\PSv3"
         }
         elseif ($PSVersion -eq '4') {
             $destinationDirBinaries = "$destinationDir\PSv4"
         }
-        else {
+        elseif ($PSVersion -eq '5') {
             $destinationDirBinaries = $destinationDir
         }
+        else {
+            $destinationDirBinaries = "$destinationDir\coreclr"
+        }
 
         # build the analyzer
-        #Write-Progress "Building for framework $Framework, configuration $Configuration"
         # The Rules project has a dependency on the Engine therefore just building the Rules project is enough
         $config = "PSV${PSVersion}${Configuration}"
         try {
             Push-Location $projectRoot/Rules
-            Write-Progress "Building ScriptAnalyzer '$framework' version '${PSVersion}' configuration '${Configuration}'"
-            $buildOutput = dotnet build Rules.csproj --framework $frameworkName --configuration "${config}"
+            Write-Progress "Building ScriptAnalyzer for PSVersion '$PSVersion' using framework '$framework' and configuration '$config'"
```

Comment:
I don't see a change in variables `$config` or `$Configuration` in the script -- just want to check you don't mean `$Configuration` here.